app/routes/user.py

```diff
@@ -3,12 +3,21 @@
 """
 from fastapi import APIRouter, Response
 
-from app.models.pydantic.UserModel import PydanticUserModify
+from app.models.pydantic.UserModel import PydanticUserModify, PydanticUserCreate
 
 import app.services.UserService as UserService
 
 userRouter: APIRouter = APIRouter(prefix="/user")
 
+
+@userRouter.post("/create", status_code=201)
+async def create_user(body: PydanticUserCreate) -> str:
```

Comment:
You should provide a more documented way of returning the password.
I know it's overkill, but create a simple pydantic model that contains one field password: str.
This is for documentation purposes.